cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -4296,6 +4334,8 @@ declare namespace Deno {
      * status. */
     output(): Promise<CommandOutput>;
     /** Kills the process with given {@linkcode Deno.Signal}.
+     *
+     * Defaults to `SIGTERM` if no signal is provided.
      *
      * @param [signo="SIGTERM"]
```

Comment:
just noticed that the default value is not being rendered on the website, which it should (and as such the text you added wouldn't be necessary)